xbmc/addons/Scraper.cpp

```diff
@@ -816,6 +816,20 @@ void DetailsFromFileItem<CArtist>(const CFileItem &item, CArtist &artist)
     artist.discography.emplace_back(discoAlbum);
   }
 
+  int nVideolinks = item.GetProperty("artist.videolinks").asInteger32();
```

Comment:
1. `const int ...`
2. https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#88-variables => no hungarian notation for variable names, please.

Reply:
Done.